docker/nocross/Dockerfile

```diff
@@ -28,7 +28,7 @@ COPY server/migrations ./migrations/
 
 # build for release
 #RUN cargo build --frozen --release
-RUN cargo build
+RUN cargo build --frozen
```

Comment:
Why does release not work?

Reply:
"--release" takes about 2 times more time, and want more RAM. I have to add swap space over defaul for raspbian  on SBC with 1 GB RAM, but without "--release" this swap required for short time. With "--release" build runs more then 90 minutes on RPi 2B. That's my reason.
"--release" will work, but takes very much time on SBC.

Reply:
I'll try this build on my machine and make sure it caches and such.